app/javascript/articles/components/SaveButton.jsx

```diff
@@ -8,20 +8,23 @@ export class SaveButton extends Component {
 
     const { isBookmarked } = props;
 
-    this.state = { buttonText: isBookmarked ? 'Saved' : 'Save' };
+    this.state = {
+      buttonText: isBookmarked ? 'Saved' : 'Save',
+      hovering: false,
+    };
   }
 
   render() {
-    const { buttonText } = this.state;
+    //const { hovering } = this.state.hovering;
 
     const { article, isBookmarked, onClick } = this.props;
 
     const mouseOver = (_e) => {
```

Comment:
Use `mouseMove` instead of `mouseOver` so that we can update the `buttonText` component state thanks to the native `mousemove` DOM event.